content/en/news/releases/1.24.x/announcing-1.24.1/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.24.1
+linktitle: 1.24.1
+subtitle: Patch Release
+description: Istio 1.24.1 patch release.
+publishdate: 2024-11-25
+release: 1.24.1
+---
+
+This release note describes what is different between Istio 1.24.0 and 1.24.1.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** unconfined AppArmor annotation to the `istio-cni-node` `DaemonSet` to avoid conflicts with
+  AppArmor profiles which block certain privileged pod capabilities. Previously, AppArmor
+  (when enabled) was bypassed for the `istio-cni-node` `DaemonSet` since privileged was set to true
+  in the `SecurityContext`. This change ensures that the AppArmor profile is set to unconfined
+  for the `istio-cni-node` `DaemonSet`.
+
+- **Added** `dnsPolicy` of `ClusterFirstWithHostNet` to `istio-cni` when it runs with `hostNetwork=true` (i.e. ambient mode).
+
+- **Fixed** an issue where `istioctl install` was not working as expected on Windows.
+
+- **Fixed** an issue where merging `Duration` with an `Envoyfilter` can lead to all listeners associated attributes unexpectedly became modified
```

Comment:
```suggestion
- **Fixed** an issue where merging `Duration` with an `EnvoyFilter` can lead to all listeners associated attributes being unexpectedly modified
```

Reply:
I'm also not quite clear on whether `listeners` is intended to be possessive, plural or both - not a huge concern but someone more familiar with this change may want to re-read the note for accuracy.